web/backend/src/cli.ts

```diff
@@ -65,4 +67,28 @@ program
     console.log('Permissions removed successfully!');
   });
 
+// Imports a list of SCIPERS from a file to allow to vote on a specific election
+// the .voters.example file is available as an example
+program
+  .command('addVoters')
+  .description('Assigns a list of SCIPERs to an Election as Voters')
+  .requiredOption('-e, --election-id <char>', 'ID of the election')
+  .requiredOption('-sf, --scipers-file <char>', 'File with line-separated list of SCIPERs')
+  .action(async ({ electionId, scipersFile }) => {
+    fs.readFile(scipersFile, 'utf8', async (err: any, data: string) => {
+      if (err) {
+        console.error(err);
+        return;
+      }
+      const scipers: Array<string> = data.split('\n');
+      const policies = [];
+      for (let i = 0; i < scipers.length; i += 1) {
+        policies[i] = [scipers[i], electionId, PERMISSIONS.ACTIONS.VOTE];
```

Comment:
I would add a sanity-check here to make sure that the sciper is an actual number between 100000 and 999999, included. In case of failure, output the line number and the offending sciper.

Reply:
I'm considering an addition like: 

```
if (scipers_list[i].length !== 6) {
      console.error(`Received a SCIPER of incorrect length on line: ${i + 1} - halting!`);
      return;
    }
```

However, I'm not sure it really adds any value. 

the CLI with file input is meant for system-to-system integration; the file is exported from EPFL systems, and imported here. (In fact, we still need the file format to be adjust the script accordingly). 

On the other hand, We don't have a particular reason to validate the operator's input as this data is external to our system.

Reply:
Wouldn't be the first time I use the wrong file, or that my file has some wrong data to it. Here it's admin-controlled data, and as an admin I would like to be informed if something is wrong with my file.

Reply:
> as an admin I would like to be informed if something is wrong with my file.

@ineiti  makes sense. I added a new commit with some validation.

Reply:
Added #356 , so closing this discussion.